From d22860e56e47f8d0a3ffc9ff3c3bbf13b4cc46e9 Mon Sep 17 00:00:00 2001
From: Akihiko Odaki <aodaki@redhat.com>
Date: Mon, 30 Oct 2023 21:25:23 +0900
Subject: [PATCH 2/4] Enable igb on x86_64
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

RH-Author: 小田喜陽彦 <None>
RH-MergeRequest: 205: Enable igb on x86_64
RH-Jira: RHEL-1308
RH-Commit: [1/1] e5079378c2e674d13b70c122096997c46ccc3dbf (akihiko.odaki/qemu-kvm)

Resolves: RHEL-1308
Signed-off-by: Akihiko Odaki <akihiko.odaki@redhat.com>
---
 configs/devices/x86_64-softmmu/x86_64-rh-devices.mak | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/devices/x86_64-softmmu/x86_64-rh-devices.mak b/configs/devices/x86_64-softmmu/x86_64-rh-devices.mak
index 6d16d81296..ef82f431cb 100644
--- a/configs/devices/x86_64-softmmu/x86_64-rh-devices.mak
+++ b/configs/devices/x86_64-softmmu/x86_64-rh-devices.mak
@@ -36,6 +36,7 @@ CONFIG_IDE_CORE=y
 CONFIG_IDE_PCI=y
 CONFIG_IDE_PIIX=y
 CONFIG_IDE_QDEV=y
+CONFIG_IGB_PCI_EXPRESS=y
 CONFIG_IOAPIC=y
 CONFIG_IOH3420=y
 CONFIG_ISA_BUS=y
-- 
2.39.3

